src/storage/webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   target: "node",
   plugins: [
     new webpack.ProvidePlugin({
-      fetch: path.resolve(path.join(__dirname, ".", "fetch-shim"))
+      fetch: path.resolve(path.join(__dirname, "../", "fetch-shim"))
```

Comment:
y do we need this "../"? is it not working previously with the "."?

Reply:
have moved `fetch-shim.js` up 1 level

Reply:
ok